PyInstaller/utils/hooks/qt.py

```diff
@@ -63,14 +68,16 @@ def __getattr__(self, name):
                 import json
                 try:
                     from %s.QtCore import QLibraryInfo, QCoreApplication
-                except:
+                except BaseException:
                     print('False')
                 else:
                     # QLibraryInfo isn't always valid until a QCoreApplication is
                     # instantiated.
                     app = QCoreApplication(sys.argv)
                     paths = [x for x in dir(QLibraryInfo) if x.endswith('Path')]
                     location = {x: QLibraryInfo.location(getattr(QLibraryInfo, x))
+                                if x != 'LibraryPath' else
```

Comment:
This PySide6 API change is also reflected here

Reply:
Indeed; here the change looks correct (and necessary), because we are trying to query both `LibrariesPath` (which works the same way in PySide2 and PySide6) and `LibraryPath` (whose behavior is changed in PySide6, and now triggers an error unless you use the use the callable version).

Reply:
Or rather, `dir(QLibraryInfo)` does not return `LibraryPath` under PySide2, only PySide6. Which is why the change is needed for PySide6 and does not affect PySide2.